build.gradle.kts

```diff
@@ -43,6 +44,8 @@ dependencies {
     implementation("com.github.AMPnet:jwt:1.0.1")
 
     implementation("com.github.komputing:kethereum:0.84.1")
+    implementation("org.web3j:core:4.8.4")
+    implementation("com.squareup.okhttp3:okhttp:4.9.1")
```

Comment:
`okhttp` is needed because of some problems with `web3j`?

Reply:
Yes, it uses older version of okhttp so there were some issues with that as well.